certified-connectors/dox42/apiDefinition.swagger.json

```diff
@@ -56,6 +56,53 @@
             "x-ms-summary": "Basic accept header, leave as is!"
           }
         ]
+      },
+      "post": {
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {}
+          }
+        },
+        "summary": "dox42 Service Call POST",
+        "description": "Make an HTTP POST request to the dox42 service",
+        "operationId": "dox42_Call_POST",
```

Comment:
@dox42diegoschleis mentioned in the comment above, please use pascal case here.

Reply:
I have changed the operationId to pascal casing